projecthub/main.py

```diff
@@ -1,16 +1,13 @@
 import argparse
-from . import Config, Settings
+from .commands import config as cmd_config
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--hello", required=False)
+    sub_parsers = parser.add_subparsers(dest="cmd")
```

Comment:
Just to avoid some confusion, maybe name the subparsers as cmd_subparsers because those are the one which will "interprete" the commands entered by the user.
```suggestion
    cmd_parsers = parser.add_subparsers(dest="cmd")
    cmd_config.setup_parser(cmd_parsers.add_parser("config"))
```

Reply:
wouldn't this part be overwritten by #18?

Reply:
Yes probably, it is just in case it's not overwritten.